tools/releaseBuild/azureDevOps/templates/testartifacts.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           linux-arm { $packageName = "TestPackage-linux-arm.zip" }
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 250
250:             $result[0] | Should -BeOfType "Microsoft.PowerShell.Commands.TestConnectionCommand+TraceStatus"

```